litmus-portal/frontend/src/redux/reducers/user.ts

```diff
@@ -7,8 +7,10 @@ import createReducer from './createReducer';
 const initialState: UserData = {
   selectedProjectID: '',
   token: '',
-  username: '',
+  username: 'admin',
```

Comment:
Why aren't we initiating as empty string?

Reply:
when the welcome modal is displayed if we initiate with empty string it will show no default username to the user on the header as we have kept welcome modal over the home page so header is visible there..this is just a default start variable used for UI, it has nothing to do with any backend interaction. @Sanjay1611 cc: @rajdas98